net/etesync-server/Makefile

```diff
@@ -0,0 +1,137 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=etesync-server
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=etesync-server-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/etesync/server/archive/v$(PKG_VERSION)#
+PKG_HASH:=d0728effa898a8b7afb4ce7439e0d0fd46bc819008925f21788d7e113435b579
+
+PKG_LICENSE:=AGPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
+
+PKG_UNPACK=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+
+define Package/etesync-server
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Web Servers/Proxies
+	TITLE:=End-to-End Encrypted Secure Data Sync
+	URL:=https://www.etesync.com/
+	DEPENDS:=+nginx-ssl +python3-django-etesync-journal \
+		+python3-drf-nested-routers +python3-django-cors-headers \
+		+python3-django-restframework +python3-django +python3-light
+	USERID:=etesync=44312
+	VARIANT:=python3
+	PROVIDES:=etesync-server
+endef
+
+define Package/etesync-uwsgi
+	$(call Package/etesync-server)
+	TITLE += using uWSGI.
+	DEPENDS += +uwsgi +uwsgi-python3-plugin +uwsgi-syslog-plugin
+endef
+
+# define Package/etesync-gunicorn
+# 	$(call Package/etesync-server)
+# 	TITLE += using gunicorn.
+# 	DEPENDS += +gunicorn3
+# endef
```

Comment:
In the beginning I was not sure which one would be better, but now I am quite sure that it will be only uwsgi, thank you for pointing me to it. I will merge it :-)